WordPress/src/main/java/org/wordpress/android/util/WPMediaUtils.java

```diff
@@ -151,4 +153,47 @@ public void onCancel(DialogInterface dialog) {
         // Do not ask again
         AppPrefs.setImageOptimizePromoRequired(false);
     }
+
+    /**
+     * Given a media error returns the error message to display on the UI.
+     *
+     * @param error The media error occurred
+     * @return String  The associated error message.
+     */
+    public static String getErrorMessage(final Context context, final MediaModel media, final MediaStore.MediaError error) {
+        if (context == null || media == null || error == null) {
+            return null;
+        }
+
+        switch (error.type) {
+            case FS_READ_PERMISSION_DENIED:
+                return context.getString(R.string.error_media_insufficient_fs_permissions);
```

Comment:
Not quite sure when this could happen. maybe if the user removed the read permission on storage after the first attempt to upload the media file failed.